src/components/CreateSighting/CreateSighting.tsx

```diff
@@ -1,23 +1,30 @@
 import React from "react";
 import { useState } from "react";
 import { FormEvent } from "react";
+import { createSighting } from "../../clients/apiClient";
 import './CreateSighting.scss';
 
 export function CreateSighting() {
-    const [date, setDate] = useState<string>("");
+    const [date, setDate] = useState<Date>(new Date());
     const [photoUrl, setPhotoUrl] = useState<string>("");
     const [latitude, setLatitude] = useState<number>(0);
     const [longitude, setLongitutde] = useState<number>(0);
     const [numberOfWhales, setNumberOfWhales] = useState<number>(0);
     const [species, setSpecies] = useState<string>("");
     const [description, setDescription] = useState<string>("");
-
+    const [status, setStatus] = useState<string>("");
+    
     function handleSubmit(event: FormEvent<HTMLFormElement>) {
-        event.preventDefault()
+        event.preventDefault();
+        createSighting({ dateOfSighting: date, locationLatitude: latitude, locationLongitude: longitude, photoImageURL: photoUrl, numberOfWhales: numberOfWhales, description: description, whaleSpecies: species })
```

Comment:
Can we create this as a variable before we pass it in so it is easier to read?
```suggestion
        const newSighting = new NewSighting{
    dateOfSighting,
    locationLatitude,
    locationLongitude,
    photoImageURL,
    numberOfWhales,
    description,
    whaleSpecies,
}
        createSighting(newSighting)
```

Reply:
Ignore the indentation in the suggestion